activiti-cloud-events-adapter/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  - name: alfresco-infrastructure
+    version: 2.0.0-SNAPSHOT
+    repository: https://kubernetes-charts.alfresco.com/incubator
```

Comment:
@cristina-sirbu this will probably clash with all the other charts.. until we don't work on unifying the infrastructures it will be hard for us to accept this..